src/index.js

```diff
@@ -1,12 +1,12 @@
 /**@license
- * mout v1.1.0 | http://moutjs.com | MIT license
+ * mout v1.1.1 | http://moutjs.com | MIT license
  */
 define(function(require){
 
 //automatically generated, do not edit!
 //run `node build` instead
 return {
-    'VERSION' : '1.1.0',
+    'VERSION' : '1.1.1',
```

Comment:
I'll add it as it's overwritten anyway, but for future reference, no need to update this file as it's generated with the deployment.